packages/sdks/output-generation/index.js

```diff
@@ -41,11 +41,11 @@ const getFolderName = () => {
  */
 const getEvaluatorPathAlias = (options = {}) => {
   const pointTo = options.pointTo || 'input';
-  const format = options.format || 'ts';
+  const format = options.format;
```

Comment:
mmm this change will change the default behaviour of this function for all other SDKs. I think many of them are expecting `format` to be set to `ts` by default.

Why do you need this change? You should always be able to use an alias with an explicit path, i.e. `${folder}/index.js` or `${folder}/index.ts`

Reply:
CI seems to pass! I was worried that certain SDK TS configs would complain about extensionless imports...but if they're not complaining I guess there's no issue here

Reply:
i checked it globally it was only used with the default `ts` format in one place which I've updated.

so the issue here is if we set it to any specific format, when angular performs transformation it goes from `ts` to `mjs` which cannot resolve any hardcoded `.ts` extension to create an FESM build. If you remember this is the reason why we added `explicitImportFileExtension` in mitosis

Reply:
Ah, thanks for explaining.

> i checked it globally it was only used with the default ts format in one place which I've updated.

Actually, we use `viteOutputGenerator()` with default arguments in many places, which ends up calling this function without a `format` argument. So there are more places where it's being used.

But now that I think about it, it seems like we never needed this `format` to begin with, and we could get away with not providing it at all.

Either way, this PR is in a good state and can be merged 👍🏽

Reply:
ohh got it